doc/doxygen/headers/concepts.h

```diff
@@ -117,8 +117,8 @@
  * DoFHandler::begin_active()) and <tt>typedefs</tt> (such as
  * DoFHandler::active_cell_iterator) for managing collections of cells.
  *
- * Triangulation&nnbsp;s, DoFHandler&nnbsp;s, and hp::DoFHandler&nnbsp;s may
- * all be considered as containers of cells. In fact, the most important parts
+ * Triangulation, DoFHandler, and hp::DoFHandler may
+ * all be considered as a container of cells. In fact, the most important parts
```

Comment:
pedantic: I wrote it that way because `Triangulation` and `DoFHandler` are not containers of cells, but instances of them are.